addons/hr_recruitment/static/src/scss/hr_job.scss

```diff
@@ -53,4 +53,9 @@
             }
         }
     }
+}
+
+.kanban_state_selection_with_text_on_left_side {
```

Comment:
Always prefix odoo classes with ``o_``
Also, be more specific on the definition to avoid evaluating the rules on all the views.
Something like the following could be better. 
``` less
.o_kanban_view .oe_kanban_card {
    .o_kanban_state_with_padding {
        padding-left:7%;
        padding-bottom:5%;
    }
}
```
No need to adapt, I'm gonna do it myself ! ;)